src/azul/plugins/metadata/hca/transform.py

```diff
@@ -332,16 +335,17 @@ class Submitter(SubmitterBase, Enum):
         SubmitterCategory.external
     )
 
-    def __init__(self, id: str, title: str, category: SubmitterCategory):
+    def __init__(self, submitter_id: str, title: str, category: SubmitterCategory):
```

Comment:
By that logic it would have to be `submitter_title`, too. Please revert the rename.